src/shared/services/email/email.service.ts

```diff
@@ -0,0 +1,28 @@
+import { createTransport } from 'nodemailer';
+import { smtpConfig } from '../../configs/index';
+
+export interface IMessageToEmail_Return {
+  from: string;
+  to: string;
+  subject: string;
+  html: string;
+}
+
+export const transporter = createTransport({
+  host: smtpConfig.host,
+  port: smtpConfig.port,
+  secure: false,
+  auth: {
+    user: smtpConfig.user,
+    pass: smtpConfig.password,
+  },
+});
+
+export const messageToEmail = (toEmail: string, code: string): IMessageToEmail_Return => {
+  return {
+    from: `<${smtpConfig.user}>`,
```

Comment:
Try `Sender_name <${...}>` - this way you can attach a proper name to the sender field